src/StorageSync/StorageSync/RegisteredServer/RegisterServerCommand.cs

```diff
@@ -160,7 +190,7 @@ public override void ExecuteCmdlet()
         /// <exception cref="PSArgumentException">AfsAgentInstallerPath</exception>
         private RegisteredServer PerformServerRegistration(string resourceGroupName, Guid subscriptionId, string storageSyncServiceName)
         {
-            using (ISyncServerRegistration syncServerRegistrationClient = new SyncServerRegistrationClient(StorageSyncClientWrapper.StorageSyncResourceManager.CreateEcsManagement()))
+            using (ISyncServerRegistration syncServerRegistrationClient = StorageSyncClientWrapper.StorageSyncResourceManager.CreateSyncServerManagement())
             {
```

Comment:
Where is StorageSyncResourceManager initialized? 

Mainly, I want to make sure that the ServerManagedIdentityProvider is having the EnableMIChecking set to false.

Reply:
But, I guess it may be okay to leave it if applicationId = null on line 232 anyway.